src/Workspaces/Core/Portable/PatternMatching/AllLowerCamelCaseMatcher.cs

```diff
@@ -21,17 +23,22 @@ private struct AllLowerCamelCaseMatcher
         {
             private readonly bool _includeMatchedSpans;
             private readonly string _candidate;
-            private readonly StringBreaks _candidateHumps;
+            private readonly ArrayBuilder<TextSpan> _candidateHumps;
```

Comment:
Don't encode string breaks.  We're not storing them anywhere (though that might be an interesting idea).  And decoding these actually has measurable overhead.